setup.py

```diff
@@ -49,16 +49,10 @@ def get_version(version):
         'rules',
         'rules.templatetags',
         'rules.contrib',
-        'rules.compat',
     ],
 
     install_requires=[
-        # 'Django >= 1.5',
-    ],
-    tests_require=[
-        'nose',
-        'coverage',
-        'Django >= 1.5',
+        'Django',
```

Comment:
Can you please move the Django requirement back to "tests_require" so it's still optional? Rules should be able to be used standalone without Django.

Reply:
Right. Should I just remove it entirely then? The `tests_require` isn't used by tox.

Reply:
Sorry - to be clear, I'm suggesting that `tests_require` isn't necessary, since tests are ran through `tox`, not `python setup.py test`.

Reply:
Gotcha. Let’s remove it completely then.